lib/sycamore/sycamore/transforms/partition.py

```diff
@@ -488,6 +490,82 @@ def partition(self, document: Document) -> Document:
         return document
 
 
+class ArynPartitioner(Partitioner):
```

Comment:
Let's update the unit/integ tests. Also, are we still keeping around SycamorePartitioner?

Reply:
We should definitely keep `SycamorePartitioner` around. The question is whether we keep the `model_server_endpoint` part of the `SycamorePartitioner` around.

Reply:
I don't think we ever expect the model_server_endpoint to be used at this point, do we? There's no way for external people to call it right? You don't need to remove it as part of this PR, but I do think we should remove it.